test/e2e_node/runner/remote/run_remote.go

```diff
@@ -324,7 +320,7 @@ func main() {
 	running := 0
 	for shortName := range gceImages.images {
 		imageConfig := gceImages.images[shortName]
-		fmt.Printf("Initializing e2e tests using image %s.\n", shortName)
+		fmt.Printf("Initializing e2e tests using image %s/%s/%s.\n", shortName, imageConfig.project, imageConfig.image)
```

Comment:
should be using klog, not printf

this solves 2.) ?

Reply:
Yes, this will give clear idea which image is used.  Output example is:
ubuntu/project/ubuntu-18-04 or
cos-stable/cos-cloud/cos-77-12347-237-0